app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -81,10 +80,9 @@ class EventsFragment : Fragment() {
 
         eventsViewModel.progress.observe(this, Observer {
             it?.let {
-                if (showRefreshLayoutProgress)
+                Utils.showProgressBar(rootView.progressBar, it)
+                if (!it)
```

Comment:
Because it shows up automatically and we only need to disable it.
Is it the ideal behaviour to show 2 progress bars at the same time?
That's why I have added this if statement.

Reply:
No, only refresh bar should be shown

Reply:
Also, there's no harm in setting it to true, if it is visible, it will remain visible